src/easy-peasy.js

```diff
@@ -4,15 +4,25 @@ import {
   createStore as reduxCreateStore,
 } from 'redux'
 import memoizeOne from 'memoize-one'
-import produce from 'immer'
+import produce, { setAutoFreeze } from 'immer'
 import thunk from 'redux-thunk'
 import { isStateObject } from './lib'
 
-const effectSymbol = '__effect__'
-const selectSymbol = '__select__'
-const selectDependenciesSymbol = '__selectDependencies__'
-const selectStateSymbol = '__selectState__'
-const reducerSymbol = '__reducer__'
+setAutoFreeze(false)
+
+const effectSymbol = Symbol('__effect__')
+const selectSymbol = Symbol('__select__')
+const selectDependenciesSymbol = Symbol('__selectDependencies__')
+const selectStateSymbol = Symbol('__selectState__')
+
+class reducerContainer {
```

Comment:
Whats the thinking behind having this container?

Reply:
Compatibility with redux-persist that use Object.freeze api and then lead to an error whan easy-peasy try to modify it, even with adding a Symbol